presto-docs/src/main/sphinx/plugin/native-sidecar-plugin.rst

```diff
@@ -0,0 +1,105 @@
+To enable sidecar support on the coordinator, add the following properties to your coordinator configuration:
+
+============================================ ===================================================================== ==============================
+Property Name                                Description                                                           Value
+============================================ ===================================================================== ==============================
+``coordinator-sidecar-enabled``              Enables sidecar in the coordinator                                    true
+``native-execution-enabled``                 Enables native execution                                              true
+``presto.default-namespace``                 Sets the default function namespace                                   `native.default`
+============================================ ===================================================================== ==============================
+
+Sidecar worker properties
+^^^^^^^^^^^^^^^^^^^^^^^^^
+Enable sidecar functionality with:
+
+============================================ ===================================================================== ==============================
+Property Name                                Description                                                           Value
+============================================ ===================================================================== ==============================
+``native-sidecar``                           Enables a sidecar worker                                              true
+``presto.default-namespace``                 Sets the default function namespace                                   `native.default`
+============================================ ===================================================================== ==============================
+
+Regular worker properties
+^^^^^^^^^^^^^^^^^^^^^^^^^
+For regular workers (not acting as sidecars), configure:
+
+============================================ ===================================================================== ==============================
+Property Name                                Description                                                           Value
+============================================ ===================================================================== ==============================
+``presto.default-namespace``                 Sets the default function namespace                                   `native.default`
+============================================ ===================================================================== ==============================
+
+
+The Native Sidecar plugin allows modular configuration, enabling users to selectively activate individual components such as the function registry, session property provider, type manager, or the plan checker.
```

Comment:
I would emphasize that these configuration properties should be specified in conjunction with each other.  I wouldn't emphasize the modular nature--I think the default is, they should all be enabled, so we want that highlighted in the user documentation.